docs/just-for-testing.apib

```diff
@@ -0,0 +1,104 @@
+FORMAT: 1A
+HOST: http://localhost:8000
+
+# History
+This service is intended to provide an easy way to consume data associated to
+device attributes, such as temperature readinds, and notifications sent by
+other services.
+
+
+## Retrieve data from attributes [/device/{device_id}/history?lastN={lastN}&attr={attr}&dateFrom={dateFrom}&dateTo={dateTo}]
+
+This endpoint retrieves the last values associated to a device. The query
+string might select more than one attribute. For instance:
+`?lastN=3&attr=temperature,pressure` will return the last 3 values from both
+`temperature` and `pressure` attributes.
+
+If no query string is set, then all values from all attributes are returned.
+
++ Parameters
+    + device_id:labtemp (required, string) - Device identifier
+    + attr:temperature (optional, string) - Device attribute to be
+    + dateFrom:`2018-06-05T18:00:00Z` (optional, string) - Start time of a time-based query
+    + dateTo:`2018-06-15T18:00:00Z` (optional, string) - End time of a time-based query
```

Comment:
In the `history.apib` file you changed the dates and the device ID. Wouldn't it be better to keep these files in sync?